EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -1206,297 +1206,297 @@ QUEST_LV_0300_20160310_001205	What do we do with the useless scriptures?{nl}We u
+QUEST_LV_0300_20160310_001302	Good. You should do it.{nl}I was actually going to Veyrhas Hall to burn the scripture...{nl}
+QUEST_LV_0300_20160310_001303	It is better to see it once than to hear it a hundred times.{nl}Let's go together. Brother.
+QUEST_LV_0300_20160310_001304	I didn't know that one of people of the Winterspoon family came to Nahash Forest.{nl}Even so, since you know this person, he or she must be reliable.{nl}I am counting on you.
+QUEST_LV_0300_20160310_001305	I can't exactly remember what I did back then.{nl}As if I was possessed by something...{nl}
+QUEST_LV_0300_20160310_001306	Yes! {nl}I wouldn't have done such an impious thing with a conscious mind.
+QUEST_LV_0300_20160310_001307	I will apologize to the other monks later.{nl}I can't believe what I did.{nl}
+QUEST_LV_0300_20160310_001308	Look around you.{nl}Don't you think it's beautiful?{nl}
+QUEST_LV_0300_20160310_001309	The red blood of the demons contains special powers.{nl}That's why only their blood can wash away the polluted power of the goddess.
+QUEST_LV_0300_20160310_001310	The Tree of Truth sect turned the monastery into a complete mess...
+QUEST_LV_0300_20160310_001311	We have to clean up the blood of the demons and tidy up the scripture...{nl}We have many things to do.
+QUEST_LV_0300_20160310_001312	It will be effective!{nl}Even so... If it goes wrong, you should tell me... For sure...
+QUEST_LV_0300_20160310_001313	Revelator, I am gratefully thankful to you.
+QUEST_LV_0300_20160310_001314	I've opened my new eyes because of Dominikas.{nl}Only the Divine Tree can save us...
+QUEST_LV_0300_20160310_001315	I need to save my power since we don't know when {nl}the Tree of Truth sect will rush into the monastery.{nl}I really thank you.
+QUEST_LV_0300_20160310_001316	How could we lose the monastery to the heresy like the Tree of Truth sect...{nl}I feel sorry to the goddess and I am thankful to her.
+QUEST_LV_0300_20160310_001317	I won't be deceived by the lies of the goddess anymore.{nl}You also think so, right?
+QUEST_LV_0300_20160310_001318	You really are...{nl}You might be the envoy that was sent by the goddess...{nl}
+QUEST_LV_0300_20160310_001319	What's strange is that Dominikas often visited Pagarba Chapel.{nl}It could very well mean that he may have hidden something important there.{nl}
+QUEST_LV_0300_20160310_001320	If you hide there, you will definitely manage to ambush Dominikas.{nl}I will pray for you in the name of the goddess.
+QUEST_LV_0300_20160310_001321	As we cleaned up the monastery...{nl}I have a bunch of the Soul Spheres that were left by Dominikas.{nl} 
+QUEST_LV_0300_20160310_001322	Can you remember when you broke the evil idol?{nl}How about I put the demon souls into the sphere and purify them.{nl}
+QUEST_LV_0300_20160310_001323	I will give you the Soul Sphere if you could defeat the demons with these, please.{nl}You can put the souls into this sphere.
```

Comment:
Is he offering "Soul Spheres" as a reward? or is he giving them as a tool to fulfill his request?